docs/release-notes/fall-2021/README.md

```diff
@@ -25,12 +25,22 @@
 | `@rei/cedar` | ^11.x.x |
 | `@rei/cdr-tokens` | ^10.x.x |
 | `@rei/cdr-component-variables` | ^7.x.x |
-| `@rei/cedar-icons` | ^2.2.x |
```

Comment:
I think having icons listed here has confused some people in to thinking they need to be importing it in to their projects (99% of time they don't).